ansible/projects/tasks/get_worker_nodes.yml

```diff
@@ -0,0 +1,7 @@
+- name: get docker containers
+  shell:
+    cmd: docker ps -f name=worker --format "{{ '{{' }}.Names{{ '}}' }}"
```

Comment:
Use ` kind get  nodes --name <cluster>`

Reply:
Ovn-kubernetes ansible is doing same task, https://github.com/Mellanox/kubernetes-ci/blob/master/ansible/projects/ovn-kubernetes/tasks/deploy.yaml#L3 , can you make both read the same file to not redundant it